pkg/server/server.go

```diff
@@ -91,12 +93,23 @@ func New(configuration options.Configuration) (controllerruntime.Manager, error)
 		return nil, err
 	}
 
+	// If configuration has a TargetLabelSelector, then read the values
+	// and create the labels.Selector object.
+	var targetLabelSelector labels.Selector
+	logging.Logger.Info("Configuration targetLabelSelector text is: " + configuration.TargetLabelSelector)
```

Comment:
we use structured logging, correct form would be 
```
logging.Logger.Info("Initializing target label selector", "label_selector", configuration.TargetLabelSelector)
```
or similar

Reply:
Thanks @grzesuav . I have updated this to use structured logging as you suggested.